astro_2.0/components/AppHeader/components/NotificationsBell/NotificationsBell.tsx

```diff
@@ -21,6 +22,7 @@ export const NotificationsBell: VFC<NotificationsBellProps> = ({
   className,
 }) => {
   const rootRef = useRef(null);
+  const [open, setOpen] = useState(false);
```

Comment:
We don't need to track open state here as we removed dropdown from this component